### ISSUE TITLE
Mod keeps crashing game

### ISSUE THREAD

**Issue by Haxxpost** [Drillgon200/Hbm-s-Nuclear-Tech-GIT#225]
I keep trying to open MC with the mod and it keeps crashing the game, not allowing to to open.

**Comment by abel1502** [comment]
This sounds like a conflict with other mods. Try a fresh run with just hbm. If it works, the problem was indeed in a conflict. If it doesn't, try running it without mods at all - maybe there's a problem with your minecraft installation

**Comment by Haxxpost** [comment]
Yes, it works with out mods. I want to keep them, though. How do I fix it without removing the mods

**Comment by abel1502** [comment]
Well, the next step is to pinpoint, which specific mod is causing conflicts. The fastest way to do so is disabling half the mods at a time. (hbm should alwaus stay enabled though). If that half fails, the problem is in one of the mods in it, otherwise, it's one of the disabled ones. You then disable everything that you know are sure of, and repeat this process with the suspect half, until you're left with one mod.
Then you can report the incompatibility here and on the other mod's tracker

**Comment by Haxxpost** [comment]
Thank you!

**Comment by Drillgon200** [comment]
It would be nice to have a crash report or a log.
